package.json

```diff
@@ -44,18 +45,18 @@
   ],
   "license": "MIT",
   "files": [
-    "src",
-    "lib",
     "scripts",
-    "docs"
+    "docs",
+    "lib"
   ],
   "dependencies": {
     "bin-build": "^3.0.0",
     "is-valid-path": "^0.1.1",
     "joi": "^17.4.0",
```

Comment:
Can we remove joi after this refactor?  or is it still used to validate something?